RELEASE_NOTES.md

```diff
@@ -7,6 +7,18 @@ in the release. For a detailed list of changed components, refer to
 This document contains release notes for the current major release and all patches.
 For prior releases, see [PRIOR\_RELEASE\_NOTES.md](PRIOR_RELEASE_NOTES.md).
 
+## 12.15.X (TBD)
+
+* [Stricter validation of non-functional user record fields](https://github.com/chef/chef-server/pull/1294),
```

Comment:
My only worry about this release note text is that it may imply that we currently have a problem with SQL injection even though we don't think that is the case.